src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -61,8 +52,6 @@ import { useShowHide } from "hooks/useShowHide";
 import { useSovereignVenue } from "hooks/useSovereignVenue";
 import { useUser } from "hooks/useUser";
 
-import { PlayaContainer } from "pages/Account/Venue/VenueMapEdition";
```

Comment:
i guess you can remove PlayaContainer too if it's not used anywhere.

Reply:
removed @ c9e363a5ad5f58002856101536d76b824738f161